docs/src/doc/decisions-and-ADRs/01_overview.stories.mdx

```diff
@@ -16,7 +16,12 @@ There is also some additional - very tech heavy - documentation contained in sub
 
 ### Packages
 
-The code of Wikit is organized into multiple packages, each tending to its own distinct task:
+The code of WiKit is organized into multiple packages, each tending to its own distinct task:
 
 * docs - the docs you are looking at, this includes general documentation and demoing the other packages
 * tokens - the Design Tokens which Wikit, as a Design system, is based on
+* vue-components - the component library that implements the Design System
+
+### Vue components
+
+The library assumes it is going to be used in a Vue project, therefor Vue is externalized.
```

Comment:
Another Nitpick: I'm not fully convinced this needs a full section, if the only purpose is to mention that Vue is externalized. We should cosider if this is redundant or maybe just add it as an (*) asterisk.

Reply:
This already existed in another part of the docs, I just moved it to the appropriate section.
We wrote it, because usually when creating a Vue app we can externalize Vue in the config (because it's available in MediaWiki core) and the code documented itself, see [data-bridge](https://gerrit.wikimedia.org/r/plugins/gitiles/mediawiki/extensions/Wikibase/+/refs/heads/master/client/data-bridge/vue.config.js#7).
But here [`vue-cli` does that for us ](https://cli.vuejs.org/guide/build-targets.html#library)when we register the project as a library, so technically it's not documented anywhere.

Reply:
How is the content of these two sections (Packages and Vue components) related to the ADRs (e.g. are they relevant extracts, or relevant information that hasn't been capture in the ADR format)? Should we clarify that here?

At the same time, this feels like Getting started material 🤔